lib/skate_web/controllers/stop_controller.ex

```diff
@@ -11,4 +11,27 @@ defmodule SkateWeb.StopController do
 
     json(conn, %{data: stations})
   end
+
+  @spec index(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  @doc """
+  Get all stops
+  """
+  def index(conn, _params) do
+    none_match = List.first(Plug.Conn.get_req_header(conn, "if-none-match"))
+    version_fn = Application.get_env(:skate_web, :version_fn, &Schedule.version/0)
+
+    etag = String.replace(version_fn.(), ",", "-")
+
+    if etag === none_match do
+      send_resp(conn, 304, "")
```

Comment:
__suggestion__(non-blocking)
```suggestion
      send_resp(conn, :not_modified, "")
```